.github/workflows/run_cell-type-wilms-tumor-06.yml

```
@@ -34,32 +34,28 @@ jobs:
   run-module:
     if: github.repository_owner == 'AlexsLemonade'
     runs-on: ubuntu-latest
+    container: public.ecr.aws/openscpca/cell-type-wilms-tumor-06:latest
+    defaults:
+      run:
+        shell: bash -el {0}
 
     steps:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Set up R
-        uses: r-lib/actions/setup-r@v2
-        with:
-          r-version: 4.4.0
-          use-public-rspm: true
-
-      - name: Set up pandoc
-        uses: r-lib/actions/setup-pandoc@v2
-
-      - name: Install system dependencies
+      - name: Install aws-cli to support the data download
         run: |
-          sudo apt-get install -y libcurl4-openssl-dev libhdf5-dev libglpk40 jags
-
-      - name: Set up renv
-        uses: r-lib/actions/setup-renv@v2
-        with:
-          working-directory: ${{ env.MODULE_PATH }}
-
-      # Update this step as needed to download the desired data
-      - name: Download test data
-        run: ./download-data.py --test-data --format SCE --projects SCPCP000006
+          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install
+
+      - name: Download test data in SCE format
```

My general feeling is that these names are meant for tagging the steps in the display, not so much as comments, so leaving them shorter (and consistent) is probably better. (and `--format SCE` is pretty clear?)
```suggestion
      - name: Download test data
```